.nvmrc

```diff
@@ -1 +1 @@
-18.18.1
\ No newline at end of file
+20.11
```

Comment:
👍 just curious what prompted this? Turnkey sdk?

Reply:
a few reasons:
- the turnkey sdk + node 18 results in test failures
- [mise](https://github.com/ethereum-optimism/actions/blob/main/mise.toml#L2) which runs our CI tests uses node 20.11
- node 18 is [no longer supported](https://www.herodevs.com/blog-posts/node-js-18-end-of-life-what-it-means-and-how-to-prepare)